src/components/layouts/LoginPage.tsx

```diff
@@ -3,22 +3,25 @@ import { Button, Card, Form, Input, message } from 'antd';
 import { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 
+interface LoginProps {
```

Comment:
Please replace the interface name with `Credentials` because we use `Props` for components like `ComponentNameProps` or `Props`